vllm/v1/engine/core.py

```diff
@@ -65,6 +66,11 @@ def __init__(
         self.structured_output_manager = StructuredOutputManager(vllm_config)
 
         # Setup scheduler.
+        if isinstance(vllm_config.scheduler_config.scheduler_cls, str):
+            Scheduler = resolve_obj_by_qualname(
+                vllm_config.scheduler_config.scheduler_cls)
```

Comment:
I'd prefer to have a log (INFO or WARNING) saying that we are using a plugged scheduler and we are not responsible for maintaining the compatibility because this is not a public interface.

Reply:
Sure!